### PR TITLE
add function updateUserIdsAndPhoneNumbers

### PR DESCRIPTION
**Pull request by xiaozdezhu**
I would like to check if I correctly understand this function or not first.

I will add test later.

### CODE REVIEW

**Comment by apsdehal** [comment]
Travis build is not failing because it is missing tests (I can merge this without tests), but because it is not compiling. See https://travis-ci.org/yoav-zibin/NewGamePortal/builds/350638442

File: src/services/firebase.test.ts [hunk 1]
```diff
@@ -112,6 +112,47 @@ it('fetch match list from firebase', done => {
...
+  // get string from contact and convert them to string
+  const phoneNumbers: string[] = [];
+  phoneNumbers.push(testPhoneNumber);
+  ourFirebase.updateUserIdsAndPhoneNumbers(phoneNumbers);
+  // check if store has been updated
+  store.subscribe(() => {
+    const newMap = store.getState().userIdsAndPhoneNumbers;
+    console.log('store:' + store.getState().userIdsAndPhoneNumbers);
+    const phoneNumberToUserId = newMap['phoneNumberToUserId'];
+    if (phoneNumberToUserId[testPhoneNumber]) {
```
Comment (apsdehal) [inline src/services/firebase.test.ts]: Seems like you are missing some code here.

File: src/services/firebase.test.ts [hunk 2]
```diff
@@ -112,6 +112,47 @@ it('fetch match list from firebase', done => {
...
+
+it('pingOpponentsInMatch', done => {
```
Comment (apsdehal) [inline src/services/firebase.test.ts]: weird, I believe this code has been merged and was written by Sisi. Must be issue with your merge, but ignore.

Comment (xiaozdezhu) [reply src/services/firebase.test.ts]: Yes, it reminds me that there is a conflict, so I resolved conflicts by myself just now. :)

File: src/services/firebase.test.ts (hunk 1 above)
Comment (xiaozdezhu) [reply src/services/firebase.test.ts]: What I want to do in if block is to check if phoneNumberToUserId has key testPhoneNumber or not. I google it and find this way.

**Comment by yoav-zibin** [comment]
I fixed the tests, you can look here:
https://github.com/yoav-zibin/NewGamePortal/commit/30ea263cb25abca00315b10acd2b9065080b58bf

**Pull request merged**